docs/sources/README.md

```diff
@@ -12,6 +12,7 @@ To add a source, add its Connector ID to the `enabled_connectors` list in your `
 | `azure-ad`                 | Azure Active Directory              | API  | DEPRECATED   |
 | `badge`                    | Badge                               | Bulk | GA           |
 | `chatgpt`                  | ChatGPT                     | API  | BETA         |
+| `confluence`               | Confluence Cloud                    | API  | BETA         |
```

Comment:
also, make sure you add to `SUMMARY.md`, so shows up in links in gitbook

and symlinke from `docs/sources/` --> `confluence/`, as we have for other sources.